src/resources/css/themes/Rounded/BaseTrack.css

```diff
@@ -96,11 +96,12 @@ BaseTrackView
    font-size: 8px; /*db value */
 }
 
-BaseTrackView[unlighted="true"]             /* track when is not xmiting */
+BaseTrackView[unlighted="true"],                            /* the track */
+TrackGroupView[unlighted="true"] > #topPanel                /* the track group top panel: used to show user name */
```

Comment:
@jonjamcam , this change will affect the topPanel background, not the tracks background. This is correct? Is your intention?